src/inputs/checkbox-radio.scss

```diff
@@ -43,7 +43,7 @@
       background-color: t(iui-color-background-1);
     }
 
-    > svg {
+    > .iui-icon {
```

Comment:
This is under `.iui-checkbox-checkmark` and `.iui-radio-dot` both of which should only have one svg inside. So we can revert

Reply:
Changed to `svg`.